estate/models/estate_property.py

```diff
@@ -0,0 +1,35 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "This is a estate property"
+    _order = "sequence"
+
+    name = fields.Char('Name', required=True)
```

Comment:
```suggestion
    name = fields.Char(string='Name', required=True)
```
Note that what you did is totally equivalent to this ☝️ . Some reviewers prefer that we name all parameters explicitly.
Also note that by default, Odoo will use the variable name to create a label if not provided.
In this case it will create "Name", so you don't really need to specify it.

Reply:
> Note that what you did is totally equivalent to this ☝️ . Some reviewers prefer that we name all parameters explicitly. Also note that by default, Odoo will use the variable name to create a label if not provided. In this case it will create "Name", so you don't really need to specify it.

okay noted !, i changed it to `Title` in local now to match the screenshots